src/main/java/seedu/duke/storage/Storage.java

```diff
@@ -1,7 +1,139 @@
 package seedu.duke.storage;
 
+import seedu.duke.record.Patient;
+import seedu.duke.storage.PatientList;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
 public class Storage {
-    public Storage() {
 
+    private String appointmentListSaveLocation;
+    private String patientListSaveLocation;
+    private PatientList patientList = new PatientList();
+    private AppointmentList appointmentList = new AppointmentList();
+
+    private static final String APPOINTMENT_LIST_SAVE_DIRECTORY = "saves";
+    private static final String APPOINTMENT_LIST_SAVE_FILEPATH = "saves/appointments.txt";
+    private static final String PATIENT_LIST_SAVE_DIRECTORY = "saves";
+    private static final String PATIENT_LIST_SAVE_FILEPATH = "saves/patients.txt";
+
+    public Storage(String filePathApptList, String filePathPatientList) {
+        this.appointmentListSaveLocation = filePathApptList;
+        this.patientListSaveLocation = filePathPatientList;
     }
+
+    //load save file for Appointments list
+    public List<Appointment> loadSavedAppointments() throws FileNotFoundException {
```

Comment:
I don't know why using List here? Isn't ArrayList still usable?

Reply:
Per Justin, helps us if we need to refactor ArrayList down the line. 

https://stackoverflow.com/questions/9852831/polymorphism-why-use-list-list-new-arraylist-instead-of-arraylist-list-n